Formula/b/biome.rb

```diff
@@ -0,0 +1,37 @@
+class Biome < Formula
+  desc "Toolchain of the web"
+  homepage "https://biomejs.dev/"
+  url "https://github.com/biomejs/biome.git",
+    tag:      "cli/v1.2.0",
+    revision: "1897c244ae26561bb9aa49c85a7ddd1d3353e417"
```

Comment:
```suggestion
  url "https://github.com/biomejs/biome/archive/refs/tags/cli/v1.2.2.tar.gz"
  sha256 "744847d50e716a5ada7eea2f9862fdad6c5321c5aa3ad15c3169e170b2fa09e7"
```

Reply:
Thanks for the suggestion. Is this the generally preferred way ? I'm assuming it's because it saves from cloning the repo, which would take longer.